### ISSUE TITLE
DANCE

### ISSUE THREAD

**Issue by rcannood** [openproblems-bio/neurips2021_multimodal_topmethods#16]
This is a continuation of PR #14. 

Thanks @wehos ! I now managed to run the test.sh script (I had to make a small change to the `test.sh` script to get the GEX2ATAC working).

The results are now:
```json
[{"ADT2GEX":0.3264,"ATAC2GEX":0.2403,"GEX2ADT":0.4083,"GEX2ATAC":0.1776,"Overall":0.2881}]
```

The original scores were:
```json
[{"ADT2GEX":0.3261,"ATAC2GEX":0.2403,"GEX2ADT":0.3898,"GEX2ATAC":0.1776,"Overall":0.2835}]
```

- [ ] Can you check whether there are differences in parameter settings which result in a worse score for GEX2ADT?
- [ ] In addition, how would you like to name your method? Can you fill in the 'name', 'description' and 'method_label' section of the yaml? See [config.vsh.yaml#L2-L8](https://github.com/openproblems-bio/neurips2021_multimodal_topmethods/blob/main/src/predict_modality/methods/DANCE/run/config.vsh.yaml#L2-L8)
- [ ] Can you create a README for your method?

Kind regards,
Robrecht

**Comment by wehos** [comment]
For the first question, I think you also need to note that the pre-trained model files you published in the debug_dance_pretrained folder on s3 should achieve a performance score of 0.3916 in GEX2ADT task. I guess the 0.4083 is another set of pre-trained models you trained from the beginning, we shouldn't just look at one bad result. 

Meanwhile, the result is within our expectations. We don't have a fixed random seed, so each time the model would be different when trained from the beginning. In the competition, the final ensemble model consisted of single models 
 which have been individually analyzed and tested, so the performance is great. I don't think the 0.4083 you listed here is a very typical result, usually the RMSE loss is lower than 0.395. But randomness does exist here.

Added on Feb.9: We continued to work on this task and now we have improved our model a lot, which can now achieve a higher score (0.3812 for GEX2ADT task) with much better robustness. We will release our codes in a new repo soon.
